client/my-sites/current-site/index.jsx

```diff
@@ -79,11 +80,7 @@ const CurrentSite = React.createClass( {
 	},
 
 	getSelectedSite: function() {
-		if ( this.props.sites.get().length === 1 ) {
-			return this.props.sites.getPrimary();
-		}
-
-		return this.props.sites.getSelectedSite();
+		return this.props.selectedSite || this.props.sites.getPrimary();
```

Comment:
I decided to rewrite the logic here as I think this makes a little more sense now that we're looking for `props.selectedSite`. There seems little use in using `getPrimary` when we already have a value for the `selectedSite`.
Correct me if I'm wrong though.